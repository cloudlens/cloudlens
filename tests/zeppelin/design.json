{"paragraphs":[{"text":"%cl\nsource(\"local:tests/zeppelin/design.log\")","dateUpdated":"Jul 28, 2016 6:02:56 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319615_-1806251198","id":"20160728-180159_1439769672","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>message</font>:&quot;Hello&quot;\t\n%html <font color=blue>message</font>:&quot;Bonjour&quot;\t\n%html <font color=blue>message</font>:&quot;Ola&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>message</font>:&quot;Hello&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;Bonjour&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;Ola&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>message</font>:&quot;Hello&quot;",""],["%html <font color=blue>message</font>:&quot;Bonjour&quot;",""],["%html <font color=blue>message</font>:&quot;Ola&quot;",""]]},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:682","dateFinished":"Jul 28, 2016 6:02:56 PM","dateStarted":"Jul 28, 2016 6:02:56 PM","focus":true},{"text":"%cl\n\n{\n  print(\"-- init\")\n}\n\nstream{\n  print(e.message)\n}\n\n {\n  print(\"-- middle\")\n}\n\nstream {\n  print(e.message)\n}\n\n {\n  print(\"-- post\")\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_444653272","result":{"code":"SUCCESS","type":"TEXT","msg":"-- init\nHello\nBonjour\nOla\n-- middle\nHello\nBonjour\nOla\n-- post\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:683"},{"text":"%cl\n\nmatch{\n    \"(?<french>Bonjour)\"\n}\n\nstream(entry) when (entry.message) {\n  print(entry.message)\n}\n\n{\n  print(\"-- Search french\")\n  frenchCount = 0\n}\n\nstream(entry) when (entry.french){\n  frenchCount += 1\n}\n\n{\n  print(\"Found \" + frenchCount + \" french!\")\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_554474109","result":{"code":"SUCCESS","type":"TEXT","msg":"Hello\nBonjour\nOla\n-- Search french\nFound 1 french!\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:684"},{"text":"%cl\nmatch{\n    \"(?<o>[oO])\";\n    \"(?<a>[aA])\"\n}\n\nstream when (e.o e.a) {\n  print(\"w\"+ e.o + e.a + \"!!!\")\n}","dateUpdated":"Jul 28, 2016 6:03:13 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_2018059385","result":{"code":"SUCCESS","type":"TEXT","msg":"wOa!!!\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:685","dateFinished":"Jul 28, 2016 6:03:13 PM","dateStarted":"Jul 28, 2016 6:03:13 PM","focus":true},{"text":"%cl\nstream {\n  print(\"w\"+ e.o + e.a)\n}","dateUpdated":"Jul 28, 2016 6:03:16 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_619101276","result":{"code":"SUCCESS","type":"TEXT","msg":"woundefined\nwoundefined\nwOa\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:686","dateFinished":"Jul 28, 2016 6:03:16 PM","dateStarted":"Jul 28, 2016 6:03:16 PM","focus":true},{"text":"%cl\nsource(\"local:tests/zeppelin/design_json.log\", json)","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_940363782","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>hi</font>:&quot;Hello&quot;\t%html <font color=blue>message</font>:&quot;eeny meeny miny moe&quot;\t%html <font color=blue>foo</font>:{&quot;bar&quot;:42}\t\n%html <font color=blue>hi</font>:&quot;Bonjour&quot;\t%html <font color=blue>message</font>:&quot;ams tram gram&quot;\t%html <font color=blue>foo</font>:4012\t\n%html <font color=blue>hi</font>:&quot;Ola&quot;\t%html <font color=blue>message</font>:&quot;pinto pinto gorgorito&quot;\t%html <font color=blue>foo</font>:&quot;bar&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>hi</font>:&quot;Hello&quot;"},{"value":"%html <font color=blue>message</font>:&quot;eeny meeny miny moe&quot;"},{"value":"%html <font color=blue>foo</font>:{&quot;bar&quot;:42}"},{"value":""}],[{"value":"%html <font color=blue>hi</font>:&quot;Bonjour&quot;"},{"value":"%html <font color=blue>message</font>:&quot;ams tram gram&quot;"},{"value":"%html <font color=blue>foo</font>:4012"},{"value":""}],[{"value":"%html <font color=blue>hi</font>:&quot;Ola&quot;"},{"value":"%html <font color=blue>message</font>:&quot;pinto pinto gorgorito&quot;"},{"value":"%html <font color=blue>foo</font>:&quot;bar&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>hi</font>:&quot;Hello&quot;","%html <font color=blue>message</font>:&quot;eeny meeny miny moe&quot;","%html <font color=blue>foo</font>:{&quot;bar&quot;:42}",""],["%html <font color=blue>hi</font>:&quot;Bonjour&quot;","%html <font color=blue>message</font>:&quot;ams tram gram&quot;","%html <font color=blue>foo</font>:4012",""],["%html <font color=blue>hi</font>:&quot;Ola&quot;","%html <font color=blue>message</font>:&quot;pinto pinto gorgorito&quot;","%html <font color=blue>foo</font>:&quot;bar&quot;",""]]},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:687"},{"text":"%cl\nmatch upon (e.hi){\n    \"(?<o>[oO])\";\n    \"(?<a>[aA])\"\n}\n\nstream when (e.o e.a) {\n  print(\"w\"+ e.o + e.a + \"!!!\")\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_2098620936","result":{"code":"SUCCESS","type":"TEXT","msg":"wOa!!!\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:688"},{"text":"CL.loadjs(\"local:lib/pervasives.js\")","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_1280172863","result":{"code":"SUCCESS","type":"TEXT","msg":""},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:689"},{"text":"%cl\nstream(entry) when (entry){\n  match(entry, \"hi\", /(Bonjour)/, \"french\")\n}\n\nstream when (e.french){\n  print(e.message)\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_1566094006","result":{"code":"SUCCESS","type":"TEXT","msg":"ams tram gram\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:690"},{"text":"%cl\nstream(entry) when (entry){\n  match(entry, \"message\", /(ams).*(gram)/, \"ams\", \"gram\")\n}\n\nstream{\n  print(e.ams + e.gram)\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_51041787","result":{"code":"SUCCESS","type":"TEXT","msg":"amsgram\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:691"},{"text":"%cl\nsource(\"local:tests/zeppelin/design_json.log\", json)","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_1300408760","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>hi</font>:&quot;Hello&quot;\t%html <font color=blue>message</font>:&quot;eeny meeny miny moe&quot;\t%html <font color=blue>foo</font>:{&quot;bar&quot;:42}\t\n%html <font color=blue>hi</font>:&quot;Bonjour&quot;\t%html <font color=blue>message</font>:&quot;ams tram gram&quot;\t%html <font color=blue>foo</font>:4012\t\n%html <font color=blue>hi</font>:&quot;Ola&quot;\t%html <font color=blue>message</font>:&quot;pinto pinto gorgorito&quot;\t%html <font color=blue>foo</font>:&quot;bar&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>hi</font>:&quot;Hello&quot;"},{"value":"%html <font color=blue>message</font>:&quot;eeny meeny miny moe&quot;"},{"value":"%html <font color=blue>foo</font>:{&quot;bar&quot;:42}"},{"value":""}],[{"value":"%html <font color=blue>hi</font>:&quot;Bonjour&quot;"},{"value":"%html <font color=blue>message</font>:&quot;ams tram gram&quot;"},{"value":"%html <font color=blue>foo</font>:4012"},{"value":""}],[{"value":"%html <font color=blue>hi</font>:&quot;Ola&quot;"},{"value":"%html <font color=blue>message</font>:&quot;pinto pinto gorgorito&quot;"},{"value":"%html <font color=blue>foo</font>:&quot;bar&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>hi</font>:&quot;Hello&quot;","%html <font color=blue>message</font>:&quot;eeny meeny miny moe&quot;","%html <font color=blue>foo</font>:{&quot;bar&quot;:42}",""],["%html <font color=blue>hi</font>:&quot;Bonjour&quot;","%html <font color=blue>message</font>:&quot;ams tram gram&quot;","%html <font color=blue>foo</font>:4012",""],["%html <font color=blue>hi</font>:&quot;Ola&quot;","%html <font color=blue>message</font>:&quot;pinto pinto gorgorito&quot;","%html <font color=blue>foo</font>:&quot;bar&quot;",""]]},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:692"},{"text":"%cl\ngroup upon (e.message){\n    \"^.*o.*\"\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_1861611308","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;hi&quot;:&quot;Hello&quot;,&quot;message&quot;:&quot;eeny meeny miny moe&quot;,&quot;foo&quot;:{&quot;bar&quot;:42}},&quot;1&quot;:{&quot;hi&quot;:&quot;Bonjour&quot;,&quot;message&quot;:&quot;ams tram gram&quot;,&quot;foo&quot;:4012}}\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;hi&quot;:&quot;Ola&quot;,&quot;message&quot;:&quot;pinto pinto gorgorito&quot;,&quot;foo&quot;:&quot;bar&quot;}}\t","comment":"","msgTable":[[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;hi&quot;:&quot;Hello&quot;,&quot;message&quot;:&quot;eeny meeny miny moe&quot;,&quot;foo&quot;:{&quot;bar&quot;:42}},&quot;1&quot;:{&quot;hi&quot;:&quot;Bonjour&quot;,&quot;message&quot;:&quot;ams tram gram&quot;,&quot;foo&quot;:4012}}"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;hi&quot;:&quot;Ola&quot;,&quot;message&quot;:&quot;pinto pinto gorgorito&quot;,&quot;foo&quot;:&quot;bar&quot;}}"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;hi&quot;:&quot;Hello&quot;,&quot;message&quot;:&quot;eeny meeny miny moe&quot;,&quot;foo&quot;:{&quot;bar&quot;:42}},&quot;1&quot;:{&quot;hi&quot;:&quot;Bonjour&quot;,&quot;message&quot;:&quot;ams tram gram&quot;,&quot;foo&quot;:4012}}",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;hi&quot;:&quot;Ola&quot;,&quot;message&quot;:&quot;pinto pinto gorgorito&quot;,&quot;foo&quot;:&quot;bar&quot;}}",""]]},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:693"},{"text":"%cl\nsource(\"local:tests/zeppelin/design_json2.log\",json)","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_1990208838","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>status</font>:&quot;Start&quot;\t%html <font color=blue>message</font>:&quot;eeny&quot;\t%html <font color=blue>verse</font>:1\t%html <font color=blue>stanza</font>:1\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;meeny&quot;\t%html <font color=blue>verse</font>:2\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;miny&quot;\t%html <font color=blue>verse</font>:3\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;moe&quot;\t%html <font color=blue>verse</font>:4\t\n%html <font color=blue>status</font>:&quot;Start&quot;\t%html <font color=blue>message</font>:&quot;ams&quot;\t%html <font color=blue>verse</font>:1\t%html <font color=blue>stanza</font>:1\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;tram&quot;\t%html <font color=blue>verse</font>:2\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;gram&quot;\t%html <font color=blue>verse</font>:3\t\n%html <font color=blue>status</font>:&quot;Start&quot;\t%html <font color=blue>message</font>:&quot;pinto&quot;\t%html <font color=blue>verse</font>:1\t%html <font color=blue>stanza</font>:1\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;pinto&quot;\t%html <font color=blue>verse</font>:2\t\n%html <font color=blue>status</font>:&quot;&quot;\t%html <font color=blue>message</font>:&quot;gorgorito&quot;\t%html <font color=blue>verse</font>:3\t","comment":"","msgTable":[[{"value":"%html <font color=blue>status</font>:&quot;Start&quot;"},{"value":"%html <font color=blue>message</font>:&quot;eeny&quot;"},{"value":"%html <font color=blue>verse</font>:1"},{"value":"%html <font color=blue>stanza</font>:1"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;meeny&quot;"},{"value":"%html <font color=blue>verse</font>:2"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;miny&quot;"},{"value":"%html <font color=blue>verse</font>:3"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;moe&quot;"},{"value":"%html <font color=blue>verse</font>:4"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;Start&quot;"},{"value":"%html <font color=blue>message</font>:&quot;ams&quot;"},{"value":"%html <font color=blue>verse</font>:1"},{"value":"%html <font color=blue>stanza</font>:1"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;tram&quot;"},{"value":"%html <font color=blue>verse</font>:2"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;gram&quot;"},{"value":"%html <font color=blue>verse</font>:3"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;Start&quot;"},{"value":"%html <font color=blue>message</font>:&quot;pinto&quot;"},{"value":"%html <font color=blue>verse</font>:1"},{"value":"%html <font color=blue>stanza</font>:1"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;pinto&quot;"},{"value":"%html <font color=blue>verse</font>:2"},{"value":""}],[{"value":"%html <font color=blue>status</font>:&quot;&quot;"},{"value":"%html <font color=blue>message</font>:&quot;gorgorito&quot;"},{"value":"%html <font color=blue>verse</font>:3"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>status</font>:&quot;Start&quot;","%html <font color=blue>message</font>:&quot;eeny&quot;","%html <font color=blue>verse</font>:1","%html <font color=blue>stanza</font>:1",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;meeny&quot;","%html <font color=blue>verse</font>:2",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;miny&quot;","%html <font color=blue>verse</font>:3",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;moe&quot;","%html <font color=blue>verse</font>:4",""],["%html <font color=blue>status</font>:&quot;Start&quot;","%html <font color=blue>message</font>:&quot;ams&quot;","%html <font color=blue>verse</font>:1","%html <font color=blue>stanza</font>:1",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;tram&quot;","%html <font color=blue>verse</font>:2",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;gram&quot;","%html <font color=blue>verse</font>:3",""],["%html <font color=blue>status</font>:&quot;Start&quot;","%html <font color=blue>message</font>:&quot;pinto&quot;","%html <font color=blue>verse</font>:1","%html <font color=blue>stanza</font>:1",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;pinto&quot;","%html <font color=blue>verse</font>:2",""],["%html <font color=blue>status</font>:&quot;&quot;","%html <font color=blue>message</font>:&quot;gorgorito&quot;","%html <font color=blue>verse</font>:3",""]]},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:694"},{"text":"%cl\ngroup upon (e.status){\n    \"^Start\"\n}\n\n","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_600873828","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;eeny&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;meeny&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;miny&quot;,&quot;verse&quot;:3},&quot;3&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;moe&quot;,&quot;verse&quot;:4}}\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;ams&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;tram&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;gram&quot;,&quot;verse&quot;:3}}\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;pinto&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;pinto&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;gorgorito&quot;,&quot;verse&quot;:3}}\t","comment":"","msgTable":[[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;eeny&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;meeny&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;miny&quot;,&quot;verse&quot;:3},&quot;3&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;moe&quot;,&quot;verse&quot;:4}}"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;ams&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;tram&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;gram&quot;,&quot;verse&quot;:3}}"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;pinto&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;pinto&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;gorgorito&quot;,&quot;verse&quot;:3}}"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;eeny&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;meeny&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;miny&quot;,&quot;verse&quot;:3},&quot;3&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;moe&quot;,&quot;verse&quot;:4}}",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;ams&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;tram&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;gram&quot;,&quot;verse&quot;:3}}",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;status&quot;:&quot;Start&quot;,&quot;message&quot;:&quot;pinto&quot;,&quot;verse&quot;:1,&quot;stanza&quot;:1},&quot;1&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;pinto&quot;,&quot;verse&quot;:2},&quot;2&quot;:{&quot;status&quot;:&quot;&quot;,&quot;message&quot;:&quot;gorgorito&quot;,&quot;verse&quot;:3}}",""]]},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:695"},{"text":"%cl\nstream (e in log) when (e){\n  print(\"----\");\n}\n\nstream (v in log.group){\n  print(\"verse \" + v.verse + \": \" + v.message);\n}","dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_2068265423","result":{"code":"SUCCESS","type":"TEXT","msg":"----\nverse 1: eeny\nverse 2: meeny\nverse 3: miny\nverse 4: moe\n----\nverse 1: ams\nverse 2: tram\nverse 3: gram\n----\nverse 1: pinto\nverse 2: pinto\nverse 3: gorgorito\n"},"dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:696"},{"dateUpdated":"Jul 28, 2016 6:01:59 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1469743319616_-1820486908","id":"20160728-180159_973747460","dateCreated":"Jul 28, 2016 6:01:59 PM","status":"READY","progressUpdateIntervalMs":500,"$$hashKey":"object:697"}],"name":"Design","id":"2BR3T6ARY","angularObjects":{"2BRB4ZK9Z":[],"2BN98KZFM":[]},"config":{"looknfeel":"default"},"info":{}}
