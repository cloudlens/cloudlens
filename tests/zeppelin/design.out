-- init
Hello
Bonjour
Ola
-- middle
Hello
Bonjour
Ola
-- post
Hello
Bonjour
Ola
-- Search french
Found 1 french!
wOa!!!
woundefined
woundefined
wOa
wOa!!!
ams tram gram
amsgram
----
verse 1: eeny
verse 2: meeny
verse 3: miny
verse 4: moe
----
verse 1: ams
verse 2: tram
verse 3: gram
----
verse 1: pinto
verse 2: pinto
verse 3: gorgorito
